.gitignore

```diff
@@ -3,6 +3,7 @@ _jest-editor/
 /coverage/
 node_modules/
 out/
+generated-icons/
```

Comment:
good catch, please also add this to `.vscodeignore` so these files will not include in the `.vsix` file either.

Reply:
added